llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Transforms.cpp

```diff
@@ -904,19 +912,52 @@ struct LinalgExtPackOpVectorizationPass
 
   void runOnOperation() override {
     MLIRContext *ctx = &getContext();
+    // Apply tiling to make outer dims be all 1s.
+    {
+      SimpleRewriter rewriter(ctx);
```

Comment:
You could just `OpBuilder` I think.

Reply:
We need a rewriter because

1. The `scf::tileUsingSCFForOp` only accept a RewriteBase argument
2. We'll have to replace op with tiled result. see l.934 (the last statement in `func.walk`).